src/pages/signup/verify.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { FormEvent, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { CONSTANTS, API_PATH, ROUTES } from '../../constants';
+import axios, { AxiosError } from 'axios';
+import { Regex } from '../../utils/regex';
+
+const { SERVER_URL } = CONSTANTS;
+
+export interface IverifyInfo {
+   studentId: string;
+   password: string;
+}
+
+export default function SignupVerify() {
+   const navigate = useNavigate();
+
+   const [verifyInfo, setVerifyInfo] = useState<IverifyInfo>({
+      studentId: '',
+      password: '',
+   });
+
+   const [studentIdError, setStudentIdError] = useState<string | null>(null);
+   const [passwordError, setPasswordError] = useState<string | null>(null);
+
+   const verify = async (verifyInfo: IverifyInfo) => {
+      try {
+         const response = await axios.post(SERVER_URL + API_PATH.SIGNUP_VERIFY, verifyInfo);
```

Comment:
서버 베이스 url 같은 경우 `index.tsx`에 전역으로 설정되어있기 때문에 빼주셔도 됩니다!
```suggestion
         const response = await axios.post(API_PATH.SIGNUP_VERIFY, verifyInfo);
```